### ISSUE TITLE
Better detection within Laravel's .blade.php files

### ISSUE THREAD

**Issue by BrandonSurowiec** [vuejs/vue-migration-helper#14]
I ran the migration helper on a project and it is falsely detecting "Interpolation within attributes" for blade files within a Laravel project.

Thinks:
`href="{{ $url }}"` should be `:href="$url"`

I have to write it like `href="@{{ url }}"` for Vue. :)

**Comment by chrisvfritz** [comment]
I'd be open to eventually adding options, such as `--interpolation-delimiter-start` and `--interpolation-delimiter-end`. I want to finish the Vue Router rules before starting on something like that though.

**Comment by BrandonSurowiec** [comment]
@chrisvfritz Is there a way to simply dump the output to a .txt/.log file? I can sort through the false positives easier.

**Comment by chrisvfritz** [comment]
@BrandonSurowiec Definitely! I'm not sure if this works on Windows, but on OS X and Linux, you can run something like:

```
vue-migration-helper src > migration-warnings.log
```

**Comment by BrandonSurowiec** [comment]
Perfect! It works on Windows and allows me to sort through it easier. Thank you!

**Comment by masonfox** [comment]
This would be a huge help! :+1:
